### ISSUE TITLE
Option(s) that do(es) not exist anymore  

### ISSUE THREAD

**Issue by hbokh** [maintux/knife-esx#13]
The "use -template" gave me some issues:

```
Error: invalid option: --use-template
```

It seems to be ruby 1.8.7 related, with these ESX-gems:

```
esx (0.4.4, 0.4.1, 0.3.2)
knife-esx (0.3.5, 0.3.2, 0.1.3)
```

**Comment by hbokh** [comment]
No issue with ruby 1.9.3, but rbvmomi-1.8.1 gives this error

```
ERROR: RuntimeError: unknown VMODL type AnyType
```

so I installed rbvmomi-1.6.0 instead.

Using this excellent example:

http://www.tommyblue.it/2013/08/28/automagically-create-and-bootstrap-chef-nodes-on-vmware-vsphere-with-knife-esx

**Comment by sethcohn** [comment]
Could this be fixed?  rbvmomi is at 1.8.1, same error.  Downgrading to 1.6.0 works.

**Comment by raphaelbertozzi** [comment]
Wow, thanks a lot. 
I did the downgrade to version 1.6.0 and now is working. ;D
